spec/components/contents_list_spec.rb

```diff
@@ -137,6 +137,16 @@ def assert_tracking_link(name, value, total = 1)
     assert_select ".gem-c-contents-list__title", text: "Cynnwys"
   end
 
+  it "applies a custom title, if one is supplied" do
+    render_component(contents: contents_list, title: "On this page")
+    assert_select ".gem-c-contents-list__title", text: "On this page"
+  end
```

Comment:
This is probably an edge case, but with translations it's always worth checking... what happens if an empty string is passed for the `title` option? (I would expect it to default back to 'Contents' but looking at the code I can't be certain).

Reply:
@andysellick it is an edge case indeed but still that was a good hunch as you spotted an issue. 
I tested and if the `title` option is an empty string, it would indeed use the empty string as the title, which was not the intended behaviour.
I amended things slightly, so now it should fall back to 'Contents' if the `title` parameter is an empty string.

Reply:
Great, nice job.